src/components/ImportForm/SourceSection/SourceSection.tsx

```diff
@@ -55,7 +55,7 @@ export const SourceSection: React.FC<SourceSectionProps> = ({
   const label = gitOnly ? 'Git repo URL' : 'Git repo URL or container image';
 
   const [{ isGit, isRepoAccessible, serviceProvider, accessibility }, accessCheckLoaded] =
-    useAccessCheck(sourceUrl, authSecret);
+    useAccessCheck(sourceUrl, authSecret, true);
```

Comment:
Either trim the sourceUrl here or inside the utility function for every use case.

Reply:
updated